plugin-server/src/cdp/services/fetch-executor.service.ts

```diff
@@ -29,6 +31,97 @@ export const RETRIABLE_STATUS_CODES = [
     504, // Gateway Timeout
 ]
 
+const parseLiquidTemplate = (
+    template: string,
+    context: any,
```

Comment:
In future - `any` will always get a strong "NO" from a review. It indicates code that has super low reliability. We type things strongly for good reason. If you do use any make sure to add a TODO to indicate why and fix it before merging